Sources/LLVM/Module.swift

```diff
@@ -77,15 +77,14 @@ public final class Module: CustomStringConvertible {
       llvm = LLVMModuleCreateWithName(name)
       self.context = Context(llvm: LLVMGetModuleContext(llvm)!)
     }
+    self.dataLayout = TargetData(llvm: LLVMGetModuleDataLayout(llvm))
   }
 
   /// Returns the context associated with this module.
   public let context: Context
 
   /// Obtain the data layout for this module.
-  public var dataLayout: TargetData {
-    return TargetData(llvm: LLVMGetModuleDataLayout(llvm))
-  }
+  public var dataLayout: TargetData
```

Comment:
N.B. This transformation is not strictly correct.  But we currently don't allow switching targets or data layouts (whoops).  Gonna fix this in a follow-up.